spring-pulsar-spring-cloud-stream-binder/src/main/java/org/springframework/pulsar/spring/cloud/stream/binder/properties/PulsarConsumerProperties.java

```diff
@@ -41,7 +41,7 @@ public class PulsarConsumerProperties extends ConsumerConfigProperties {
 	@Nullable
 	private Class<?> messageValueType;
 
-	private Integer partitionCount = 1;
```

Comment:
I would suggest we move this back to big integer (and change the property in the binder to big integer) as that allows us to differentiate "user has not set anything" from "this is the default". In this case, because we set a default it only provides consistency w/ the other fields that have moved over to the non-primitive primitive wrappers. 

Wdyt?

Reply:
ok, good point.